### PR TITLE
fix(clients): update README snippets and contributing guides

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-2496

### Changes included:

- [x] csharp
- [x] dart
- [x] go
- [x] java
- [x] javascript
- [x] kotlin
- [x] php
- [x] python
- [x] ruby
- [x] scala
- [x] swift

update contributing and troubleshooting sections
update getting started (use https://api-clients-automation.netlify.app/docs/clients/usage)
update doc links

### CODE REVIEW

File: clients/algoliasearch-client-ruby/README.md [hunk 1]
```diff
@@ -51,12 +51,12 @@ For full documentation, visit the **[Algolia Ruby API Client](https://www.algoli
...
 ## ❓ Troubleshooting
 
-Encountering an issue? Before reaching out to support, we recommend heading to our [FAQ](https://www.algolia.com/doc/api-client/troubleshooting/faq/ruby/) where you will find answers for the most common issues and gotchas with the client.
+Encountering an issue? Before reaching out to support, we recommend heading to our [FAQ](https://www.algolia.com/doc/api-client/troubleshooting/faq/ruby/) where you will find answers for the most common issues and gotchas with the client. You can also open [a GitHub issue](https://github.com/algolia/api-clients-automation/issues/new?assignees=&labels=&projects=&template=Bug_report.md)
 
-## Upgrade from V1 to V2
+## Contributing
 
-If you were using the v1 and wish to update to v2, please follow our [Upgrade Guide](upgrade_guide.md)
+This repository hosts the code of the generated Algolia API client for Ruby, if you'd like to contribut, head over to the [main repository](https://github.com/algolia/api-clients-automation). You can also find contributing guides on [our documentation website](https://api-clients-automation.netlify.app/docs/contributing/introduction).
```
Comment (millotp) [inline clients/algoliasearch-client-ruby/README.md:58]: `/s/contribut,/contribute,`

**Review by millotp — COMMENTED** [review]
love it

File: clients/algoliasearch-client-ruby/README.md (hunk 1 above)
Comment (shortcuts) [reply clients/algoliasearch-client-ruby/README.md:58]: yessss gg

**Review by millotp — APPROVED** [review]
gg

**Pull request merged**